jina/clients/mixin.py

```diff
@@ -9,6 +10,21 @@
     from jina import DocumentArray
 
 
+def _include_results_field_in_param(parameters: Optional['Dict']) -> 'Dict':
+    if parameters:
+        if 'results' in parameters.keys():
```

Comment:
Use a protected `key`. `__RESULTS__` . Otherwise it is a too common word

Reply:
the issue with this option is that when the user is going to access the result he will have to do 
`paramaters["__results__"]` so he will probably think that he is doing something wrong